playbooks/de-deploy-services.yaml

```diff
@@ -49,6 +49,7 @@
       msg: "Deploying data-info"
     - role: de-deploy-service
       deploy_use_color: "{{use_color|default(false)}}"
+      has_configs: "{% if use_consul_configs %}{{ false }}{% else %}{{ true }}{% endif %}"
```

Comment:
Missed this the first time, but this is in the data-info deployment section, not apps. d'oh